### DISCUSSION TITLE
Reverse proxy on docker custom bridge network

### DISCUSSION THREAD

**Discussion by tiaden** [coder/coder#8574] [General] [answered]
Thank you for this amazing product. I'm having error accessing a pgadmin web app exposed as coder app. as you can see in following gist I have created a custom docker network where the pgadmin container should be visible to the coder agent. 
https://gist.github.com/tiaden/c533a2e03e4e411374ec075315542fb7
When I try to access pgadmin via the web terminal, it resolves the pgadmin container correctly 
![terminal](https://github.com/coder/coder/assets/33968665/3a616cf4-aeb5-49cc-b843-aca807cc1fe0)
Hovewer, when i try to access it through the dedicated coder app, I get the following error. Please note that i have adjusted the base path of pgadmin with no success
![coder-app-pgadmin-error](https://github.com/coder/coder/assets/33968665/c17f8aa2-8b1c-49f9-8b80-afcf3b23b2c6)
What am I doing wrong ? please advise

**Comment by matifali** [comment, marked as answer]
`coder_app` doesn't proxy non localhost addresses. The [`url`](https://gist.github.com/tiaden/c533a2e03e4e411374ec075315542fb7#file-main-tf-L40) here should be of the form http://localhost:PORT[/subpath]

**Reply by tiaden** [reply]
Ok thank you, in that case please reflect this clearly in your document here https://registry.terraform.io/providers/coder/coder/latest/docs/resources/app#url
